### ISSUE TITLE
Remove flarevm.installer.vm from MyGet

### ISSUE THREAD

**Issue by Ana06** [mandiant/VM-Packages#907]
### Details

FLARE-VM migrated to use the debloat.vm and installer.vm packages in https://github.com/mandiant/flare-vm/pull/513 and we removed the flarevm.installer.vm package from VM-Packages in https://github.com/mandiant/VM-Packages/pull/752. This package is still in myget. Removing it will break any old script still in use that relies on it (forcing to update to the newer version with great improvements and tool addition 😉). Any objections on removing flarevm.installer.vm from MyGet?

**Comment by binjo** [comment]
I agree, but can we push an update to let the users know and update to newer version?

**Comment by MalwareMechanic** [comment]
Unfortunately, there's really no "update" feature with the current VMs. @Ana06 if my memory serves, we were in favor of simply performing a fresh install versus supporting official "updates" right?

But I'm in favor of removing it

**Comment by Ana06** [comment]
Yes, we do not support updates (this is documented in the readme and the wiki) and the installer was modified in Oct 13.

**Comment by Ana06** [comment]
For other packages we could request the new package as dependency and add a message in the installation. But the installer package is special as to support the new one we changed the installer script, which mean you can't really install the old one with flare-vm anymore.
